File: Multi_Language_CodeExamples/swift/classes.swift
import Foundation

class Calculator {
    func add(_ a: Int, _ b: Int) -> Int {
        return a + b
    }
    
    func multiply(_ a: Int, _ b: Int) -> Int {
        return a * b
    }
}

let calc = Calculator()
print("5 + 3 = \(calc.add(5, 3))")
print("5 * 3 = \(calc.multiply(5, 3))")

File: Multi_Language_CodeExamples/typescript/classes.ts
class Calculator {
    add(a: number, b: number): number {
        return a + b;
    }
    
    multiply(a: number, b: number): number {
        return a * b;
    }
}

const calc = new Calculator();
console.log(`5 + 3 = ${calc.add(5, 3)}`);
console.log(`5 * 3 = ${calc.multiply(5, 3)}`);
